packages/app/src/vis-packs/nexus/hooks.ts

```diff
@@ -56,3 +60,18 @@ export function useAxisMapping(
     );
   });
 }
+
+export function useAuxiliaries(
+  auxDatasets: NumArrayDataset[],
+  selection: string | undefined
+): Auxiliary[] {
+  const { attrValuesStore } = useContext(ProviderContext);
+
+  const auxValues = useDatasetValues(auxDatasets, selection);
+
+  return auxDatasets.map((dataset) => ({
+    name: dataset.name,
+    label: getDatasetLabel(dataset, attrValuesStore),
+    value: auxValues[dataset.name],
```

Comment:
I would remove `name` and put it as a fallback of domain. I don't think it is worth tossing `name` around.

Sure, `label` can be long but we already include the labels of axes in the tooltip so why not those of signals ?